### DISCUSSION TITLE
Drupal module update as Composer package is not detected?

### DISCUSSION THREAD

**Discussion by back-2-95** [renovatebot/renovate#11142] [Request Help] [answered]
Hello, I have Drupal 7 project which is setup with Composer 2.

I see now 2 modules where there is an update (non-major) in Git source but when checking Renovate log, it does not see the new version.

There was still PRs for some modules and themes for major version updates.

This is one from Renovate logs and says pretty much that 1.6 is the current AND the latest version.

```
{
            "depType": "require",
            "depName": "drupal/search_api_autocomplete",
            "currentValue": "^1.6",
            "datasource": "packagist",
            "lockedVersion": "1.6.0",
            "depIndex": 46,
            "warnings": [],
            "versioning": "composer",
            "sourceUrl": "https://git.drupalcode.org/project/search_api_autocomplete.git",
            "homepage": "https://www.drupal.org/project/search_api_autocomplete",
            "currentVersion": "1.6.0",
            "fixedVersion": "1.6.0",
            "updates": []
          }
```

If we check the Git, there is a 1.7.0 release tag: https://git.drupalcode.org/project/search_api_autocomplete/-/tags

We have this conf for Drupal modules:

```
   {
      "matchPackagePrefixes": [
        "drupal/"
      ],
      "addLabels": [
        "drupal",
        "drupal-module"
      ]
    }
```

In composer.json we have this in `repositories` section:

```
{
            "type": "composer",
            "url": "https://packages.drupal.org/7"
        }
```

Any idea why it could not detect this change?

**Comment by rarkins** [comment, marked as answer]
Look at documentation for rangeStrategy

**Reply by back-2-95** [reply]
Hi, I was able to some what get where I wanted with rangeStrategy:bump
